src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ArgumentSyntaxExtensions.cs

```diff
@@ -49,18 +49,19 @@ public static RefKind GetRefKind(this ArgumentSyntax argument)
             bool allowParams = false,
             CancellationToken cancellationToken = default)
         {
-            if (!(argument.Parent is BaseArgumentListSyntax argumentList))
+            if (argument.Parent is not BaseArgumentListSyntax argumentList)
             {
                 return null;
             }
 
-            if (!(argumentList.Parent is ExpressionSyntax invocableExpression))
+            if (argumentList.Parent is not ExpressionSyntax &&
+                argumentList.Parent is not PrimaryConstructorBaseTypeSyntax)
             {
                 return null;
             }
```

Comment:
Currently, `TestClassBaseType` fails because I didn't add `ConstructorInitializerSyntax` to this if condition.

I'm thinking that we can just drop this if condition all together and the method should just work for everything.

Reply:
yes. i think this is sensible.